apps/aparavi-ui/src/AparaviApp.tsx

```
@@ -36,9 +36,9 @@
 import React, { useCallback, useMemo, useRef, useEffect, useState } from 'react';
 import type { CSSProperties } from 'react';
 import type { ShellAppProps } from 'shell';
-import type { IVirtualFileSystem } from 'shared/modules/explorer/types';
+import type { IVirtualFileSystem } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)